pkg/plugins/golang/v3/scaffolds/internal/templates/main.go

```diff
@@ -188,6 +188,8 @@ import (
 	"flag"
 	"os"
 
+	"k8s.io/klog/v2"
```

Comment:
Hi @astraw99, 

Thank you for the clarifications. I am OK with the change proposed/ You are not the first one looking for klog but I think would be nice to get others' opinions as well. @jmrodri @varshaprasad96 @everettraven wdyt? 

Also, I am not sure why we use 

``` 
opts := zap.Options{
		Development: true,
``` 

In the default scaffold, however, it seems that would be nice to check it out and see if we should not need to ensure the same behaviour with klog.